### DISCUSSION TITLE
Ability to duplicate mods to new AKI-instance for testing

### DISCUSSION THREAD

**Discussion by Plofkat** [SchraderR/sp-tarkov-client#140] [Ideas]
I would like to make a list of mods on my current server and apply those to another server or to another local AKI-instance.

This would make it easy to give my current mod list to a friend or just help me to test my current mods on a new AKI-instance

**Comment by SchraderR** [comment]
Something related to this? https://github.com/SchraderR/sp-tarkov-client/discussions/131
This could be extended with additional functions

**Reply by Plofkat** [reply]
That looks good! It does exactly what I need.
